tests/test_scheduler.py

```diff
@@ -102,6 +101,19 @@ def fixture_frontend():
     return FP if HAVE_FP else OFP
 
 
+def graphviz_present():
+    """
+    Test if graphviz is present and works
+    """
+    try:
+        import graphviz as gviz
+        callgraph = gviz.Graph()
+        callgraph.pipe()
```

Comment:
This could probably be shortened to gviz.Graph().pipe() since you don't care about the actual return value.

Reply:
Thanks, done